src/Plugin.php

```diff
@@ -70,6 +72,18 @@ public static function init() {
     add_action('wp_enqueue_scripts', __CLASS__ . '::wp_enqueue_scripts');
   }
 
+  /**
+   * Adds print price label capability to allowed roles.
+   *
+   * @param array $roles
+   *   List of role names allowed to print price labels.
+   */
+  public static function addPrintLabelCapability(array $roles) {
```

Comment:
There's currently no method to revoke/delete the capability. You should ensure to remove the capability when uninstalling the plugin.

Reply:
Again, nailed it.